ui/src/components/HistoryChart.svelte

```diff
@@ -0,0 +1,93 @@
+<script>
+    import {onMount} from 'svelte';
+    import { forEach, groupBy, props } from "ramda";
+
+    export let value = [];
+
+    let allScanDur = [];
+    let score = [];
+    
+    value.forEach(i => {
+        allScanDur.push(i.scanDuration)
+    })
+
+    let scanDur = allScanDur.slice(0, 10);
+
+    for (let i=0; i < 10; i++){
+        if (scanDur.length < 10) {
+            scanDur.push(5)
```

Comment:
this number need to be calculated based on the maxBarHeight,e.g.1/100 of the maxBarHeight, otherwise if maxBar height is say 20 then it will look weird,